src/main/java/org/frc5587/bunnybots/ControlClaw.java

```diff
@@ -0,0 +1,27 @@
+package org.frc5587.bunnybots;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class ControlClaw extends Command {
+
+    private Claw claw = new Claw();
+
+    public ControlClaw() {
+
+    }
+
+    public void FullControlClaw() {
```

Comment:
```suggestion
    protected void execute()() {
```
So I think that this code is what you're attempting to run every 20ms, given that you're checking for user input here, so remember that we use the execute method of commands to repeatedly run code.